conf/messages.en

```diff
@@ -207,6 +207,23 @@ view.export-questions.freightType.title=Which type of transport are you using?
 view.export-questions.freightType.heading=Which type of transport are you using?
 view.export-questions.vessel-details.title=Tell us about your vessel
 view.export-questions.vessel-details.heading=Tell us about your vessel
+view.export-questions.hasPriorityGoods.description=Priority goods require exceptional treatment. These include:
```

Comment:
The import priority stuff is under the section: # Export Questions View

By no means an issue though - rest of PR looks great & happy to approve